### PR TITLE
Moneropedia entries into Norwegian (A–B)

### PR DESCRIPTION
**Pull request by MindWallet**
Moneropedia entries into Norwegian (A–B)

### CODE REVIEW

**Comment by MindWallet** [comment]
I tried to edit my last pull request, but it seems a new one was made. Sorry about that. I will try to make the next one in my terminal, so please bear with me as I am a novice when it comes to this stuff.

File: _i18n/nb-no/resources/moneropedia/address-book.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Bruk en jump-tjeneste til å sette inn I2P-adressene i adresseboken din
+2. Bruk en jump-tjeneste til å kopiere/lime inn en adresse i ditt private abonnement
+3. Legg til eller fjern et privat abonnement manuelt
+
+**Merk: Kovri er under kontinuerlig utvikling. I fremtiden *vil* det finnes enklere måter å oppdatere adresseboken på.**
+
+### Adressebok/Navnespesifikasjon
+
+For spesifikasjonsdetaljer og mer, kan du besøke [Adressebok og navnespesifikasjon](https://geti2p.net/en/docs/naming)
```
Comment (erciccione) [inline _i18n/nb-no/resources/moneropedia/address-book.md:33]: As i mentioned in the other PR, this entry has been recently removed (#1219) and should  be removed from this PR as well.

**Review by erciccione — CHANGES_REQUESTED** [review]
The review started in #1259. Beside that looks good.

File: _i18n/nb-no/resources/moneropedia/address-book.md (hunk 1 above)
Comment (erciccione) [reply _i18n/nb-no/resources/moneropedia/address-book.md]: Fixed.

File: _i18n/nb-no/resources/moneropedia/account.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+terms: ["account", "accounts", "wallet", "wallets", "konto", "kontoer", "lommebok", "lommebøker"]
+summary: "Tilsvarende en bankkonto og inneholder alle dine sendte og mottatte transaksjoner"
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+De som kjenner til Moneros forgjengere, vil være bedre kjent med begrepet *lommebok* i denne beskrivelsen. I Monero kaller vi dette for en konto, og det er en privat konto, eid og styrt av en Monero-bruker.
+
+Kontoen din inneholder alle Monero-transaksjonene du har sendt og mottatt. Kontobalansen din er summen av alle Moneroene du har mottatt, fratrukket Moneroene du har mottatt. Når du bruker Monero, legger du kanskje merke til at kontoen din har to saldoer – en låst og en ulåst saldo. Den ulåste saldoen inneholder midler som kan brukes umiddelbart, og den låste saldoen inneholder midler som ikke kan brukes med én gang. Kanskje du motta en transaksjon som har en forhåndsinnstilt opplåsningstid, eller kanskje du har sendt noen Monero og venter på at vekslepengene kommer tilbake til lommeboken din. Begge er situasjoner som kan føre til at midler låses i en viss periode.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/account.md:11]: Kontobalanse -> Kontosaldo

File: _i18n/nb-no/resources/moneropedia/account.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Dette betyr at man enkelt kan opprette en Monero-konto for seg selv, i tillegg til en felleskonto som kan deles med en partner og individuelle kontoer for barna deres. På tilsvarende måte kan et foretak opprette separate kontoer for hver avdeling eller gruppe. Siden Moneros transaksjonskostnader er svært lave, er det rimelig å forflytte midler mellom kontoer.
+
+### Kryptografiske nøkler
+
+Monero avhenger i stor grad av et kryptografisk prinsipp kalt *kryptografi med offentlig/privat nøkkel* eller *asymmetrisk kryptografi*, som er grundig forklart i [denne Wikipedia-artikkelen](https://en.wikipedia.org/wiki/Public-key_cryptography).
+
+Kontoen din er basert på to nøkler – en forbruksnøkkel og en visningsnøkkel. Forbruksnøkkelen er spesiell i og med at det er den eneste nøkkelen som trengs for å bruke Monero-midlene dine, mens visningsnøkkelen lar deg avsløre alle transaksjonene dine til en tredjepart, for eksempel for revisjon eller bokføringsformål. Disse nøklene i kontoen din spiller også en viktig rolle i Moneros transaksjonspersonvern.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/account.md:25]: Not a mistake, but "bokføringsøyemed" sounds better.

File: _i18n/nb-no/resources/moneropedia/address.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+terms: ["address", "addresses", "adresse", "adresser"]
+summary: "enten et alias, som for eksempel donate.getmonero.org, eller et sett med 95 tegn som starter med en 4"
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/address.md:3]: Perhaps change "started med en" with "begynner med"

File: _i18n/nb-no/resources/moneropedia/airgap.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+terms: ["airgap", "luftvegg"]
+summary: "En luftvegg er et sikkerhetstiltak for å fysisk separere en PC eller enhet fra alle andre nettverk, som for eksempel internettet."
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+«En luftvegg, ‘air gap’ eller ‘air gapping’ er et sikkerhetstiltak som brukes på ett eller flere PC-er for å sikre at et sikkert PC-nettverk er fysisk isolert fra usikrede nettverk, som for eksempel det offentlige internettet eller et usikret, lokalt nettverksområde.[2] Navnet har sitt opphav fra teknikken av å skape et nettverk som er fysisk adskilt (med en konseptuell luftvegg) fra alle andre nettverk. Luftveggen bør kanskje ikke tolkes bokstavelig, i og med at nettverk som tar i bruk dediserte, kryptografiske enheter kan tunnellere pakker over upålitelige nettverk mens de unngår pakkehastighet eller størrelsesvariasjon, kan betraktes som ‘air gapped’, i og med at det ikke er noen muligheter for PC-er på motsatte side av luftveggene å kommunisere med hverandre.»
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/airgap.md:9]: ett -> én

File: _i18n/nb-no/resources/moneropedia/block.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+En blokk er en beholder av transaksjoner, der en ny blokk legges til blokkjeden hvert 2. minutt i gjennomsnitt (se konstant `DIFFICULTY_TARGET_V2` (vanskelighetsmål), som er definert som 120 sekunder).
+
+Blokker inneholder også en spesiell transaksjonstype, nemlig coinbase-transaksjonen, som legger til nylig opprettede Monero til nettverket.
+
+Blokker dannes gjennom en prosess av utvinning, og noden som suksessfullt utvinner blokken, kringkaster den til hver av nodene som er koblet til den, som deretter kringkaster blokken videre helt til hele Monero-nettverket har mottatt den.
+
+Falske eller dårlige blokker kan generelt ikke opprettes, i og med at noder som mottat blokker alltid verifiserer transaksjonene de inneholder  mot et sett med konsensusregler som alle noder føyer seg etter, blant annet validering av kryptografiske signaturer på hver transaksjon.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/block.md:15]: mottatt -> mottar

Also, I think there is an extra space after "inneholder"

File: _i18n/nb-no/resources/moneropedia/bulletproofs.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+summary: "en ny type rekkeviddebevis som tilslører det sendte beløpet ved å erstatte RingCT-er i transaksjoner"
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+RingCT-er ble introdusert for å tilsløre transaksjonsbeløpene. Ett mål med RingCT-ene var å bevise at summen av inndataene fratrukket utdataene  i transaksjonen var lik 0, og at alle utdata var positive tall. For å oppnå dette ble to typer ringsignaturer opprettet: Én ringsignatur for hele transaksjonen (for å bevise at summen er 0), og et sett med ringsignaturer for undergruppene av transaksjonsdelene (for å bevise at utdataene er positive tall), og deretter kombinere dem ved å bruke Schnorr-signaturer (som senere ble byttet ut med Borromean-ringsignaturer).
+Mens den utførte jobben, viste det seg at den omfattende størrelsen på ringCT-transaksjonene var en stor ulempe.
+
+### Når det kommer til bulletproofs
+I 2017 krev en [krypto-gruppe på Stanford](https://crypto.stanford.edu/bulletproofs/) en [vitenskapelig artikkel](https://eprint.iacr.org/2017/1066.pdf) der de presenterte en ny type rekkeviddebevis, kalt bulletproofs.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/bulletproofs.md:12]: typo: -> skrev*

File: _i18n/nb-no/resources/moneropedia/bulletproofs.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+Mens den utførte jobben, viste det seg at den omfattende størrelsen på ringCT-transaksjonene var en stor ulempe.
+
+### Når det kommer til bulletproofs
+I 2017 krev en [krypto-gruppe på Stanford](https://crypto.stanford.edu/bulletproofs/) en [vitenskapelig artikkel](https://eprint.iacr.org/2017/1066.pdf) der de presenterte en ny type rekkeviddebevis, kalt bulletproofs.  
+
+> Bulletproofs er korte, ikke-interaktive null-kunnskap-bevis som ikke krever et pålitelig oppsett.
+
+I motsetning til Borromean- eller Schnorr-signaturer, er bulletproofs veldig effektive som rekkeviddebevis. Å bevise en stort datamengde genererer kun et lite bevis, og størrelsen på disse bevisene vokser logaritmisk med datastørrelsen som bevises.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/bulletproofs.md:16]: Typo: stort -> stor

File: _i18n/nb-no/resources/moneropedia/bulletproofs.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+Å øke antallet inndata i en transaksjon med bulletproofs, øker bare størrelsen på beviset litt.    
+Bulletproofs har også fordelen ved at det kan bevise at flere dedikerte kontoer er i den ønskede rekkevidden på én gang. Det er ikke noe behov for å bevise hver utdata til hver destinasjon i separate bevis; hele transaksjonsbeløpet kan bevises i én større (men fremdeles veldig liten) bulletproof.
+
+### Gjennomgående revisjonsprosess og implementering
+I og med at bulletproofs var veldig nytt og den opprinnelige implementeringen som ble gjort av gruppen – til tross for at den ble gjort grundig – trengte den en omskriving som var fokusert på vårt spesifikke bruksområde, var det ingen enkel sak å implementere bulletproof i Monero.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/bulletproofs.md:21]: Remove "den" + typo in "omskriving"

File: _i18n/nb-no/resources/moneropedia/address.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+Når du sender Monero til noen, trenger du kun én opplysning, og det er Monero-adressen deres. En *rå* Monero-adresse er et sett med 95 tegn som starter med en '4'. Adressen til Monero-donasjonene er for eksempel <span class="long-term">888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnjyPbb3iQ1YBRk1UXcdRsiKc9dhwMVgN5S9cQUiyoogDavup3H</span>.
+
+Fordi disse adressene er lange og komplekse, vil du ofte støte på en OpenAlias-adresse i stedet. Monero-donasjoner kan for eksempel sendes til <span class="long-term">donate@getmonero.org</span> eller <span class="long-term">donate.getmonero.org</span>.
+
+Hvis du vil få din egen OpenAlias-adresse, finnes informasjon om dette på [OpenAlias-siden](/the-monero-project/).
```
Comment (erciccione) [inline _i18n/nb-no/resources/moneropedia/address.md:13]: Please adapt to #1309

```suggestion
Hvis du vil få din egen OpenAlias-adresse, finnes informasjon om dette på [OpenAlias-siden](https://openalias.org/).
```

**Review by erciccione — COMMENTED** [review]
Looks good except for one minor detail.

**Review by erciccione — APPROVED** [review]
Needs to be squashed.

**Review by erciccione — CHANGES_REQUESTED** [review]
'@' tags need to be reintroduced.

**Review by erciccione — APPROVED** [review]

**Pull request merged**